<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_MainWindow</class>
 <widget class="QMainWindow" name="UI_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Title Displayer</string>
  </property>
  <widget class="QStackedWidget" name="MainWidget">
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
